### PR TITLE
config: make pseudo estimate ratio configurable

### PR DESCRIPTION
**Pull request by lamxTyler**
If the pseudo stats is used, there won't be any dynamic update, so this PR make it configurable.
PTAL @coocood @winoros

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

File: config/config.toml.example [hunk 1]
```diff
@@ -139,6 +139,9 @@ run-auto-analyze = true
 # Probability to use the query feedback to update stats, 0 or 1 for always false/true.
 feedback-probability = 0.0
 
+# Pseudo stats will be used if the ratio between the modify count and row count of a table is greater than it.
```
Comment (zz-jason) [inline config/config.toml.example]: s/row count/row count in statistics/ ?

**Pull request merged**